### ISSUE TITLE
[Feature Request]:  Adding Sample Products Using APi calls

### ISSUE THREAD

**Issue by Ayushmaanagarwal1211** [MAVRICK-1/e-commerce_website#337]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

I will add some more products to the products section by using the fake api calls from the api .

### Use Case

This will looks more real

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

**Comment by sanket95droid** [comment]
@Ayushmaanagarwal1211 Hey can you make this sample products distinct from each other??

**Comment by Ayushmaanagarwal1211** [comment]
Yes i will fetch different api's because many api's are freely available on the internet for demo products
